### DISCUSSION TITLE
Immediate crash

### DISCUSSION THREAD

**Discussion by 44cK3r** [skyline-emu/skyline#1718] [General]
Hey sorry to bother you all again sorry but the latest version just crashed every time I try to open it up I hope I helped sorry again thank you love you all

**Comment by yhak22** [comment]
What phone ? 
Is it open in previous builds ?

**Reply by 44cK3r** [reply]
Oppo Find X2 Pro and I'll go and check what latest version works

**Reply by 44cK3r** [reply]
Ok what happened #1255 that's the one that wasn't working now The latest version which #1254 is working sorry if I didn't word anything properly I'm not good at it sorry

**Reply by yhak22** [reply]
It's ok but be sure to ask in discord what latest build to use 
cause 1255 is troll build not by devs.

**Reply by 44cK3r** [reply]
Weird how did it end up on the GitHub Actions page then

**Reply by 44cK3r** [reply]
I don't have a VIRUS DO I oh no no no no no no no

**Reply by yhak22** [reply]
Some dumbs people add requests
On the github action instead in discussions

**Comment by YapoOriginal** [comment]
I have a problem where the emulator stays black when opening any game, I think it's because of the processor ;v, they couldn't solve it, it's a Matepad Pro 12.6 inch help me pls
